### PR TITLE
Feature/sal user image

### PR DESCRIPTION
**Pull request by GreenWithMV**
## Description
Takes Sal's PR adding Gravatar and gives an option to change one's avatar if they do not make a Gravatar. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [ ] The console is clean. No errors or linting warnings

### CODE REVIEW

**Comment by SalMeza** [comment]
Sal has reviewed.

File: src/pages/Profile/Sidebar/_Sidebar.scss [hunk 1]
```diff
@@ -153,6 +153,21 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 500px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    border-bottom: 0.5px solid rgb(200, 200, 200);
+    padding: 10px 40px 20px 40px;
+    color: rgb(74, 89, 90);
+    .gravatar-info-main {
```
Comment (timimsms) [inline src/pages/Profile/Sidebar/_Sidebar.scss:167]: looks like we're not using this class so we could remove or condense to `.gravatar-info-main {}`.

File: src/pages/Profile/subpages/Settings/Basics.js [hunk 2]
```diff
@@ -9,19 +9,19 @@ const genderOptions = [
   "male",
   "female",
   "transgender female",
-  "transgender male"
+  "transgender male",
```
Comment (timimsms) [inline src/pages/Profile/subpages/Settings/Basics.js:12]: Less about this PR/code, but we may want to revisit this list based on existing standards - e.g., Facebook's: https://abcnews.go.com/blogs/headlines/2014/02/heres-a-list-of-58-gender-options-for-facebook-users

File: src/pages/Profile/subpages/Settings/Basics.js [hunk 3]
```diff
@@ -99,10 +108,30 @@ export default function Basics({
...
+        <div className="input-container">
+          <span className="input-header">Email</span>
+          <input
+            type="email"
+            id="email"
+            className="input"
+            defaultValue={email}
+            onChange={handleInput}
+          />
+          {/*errors.email && <span className="error">{errors.email}</span>*/}
```
Comment (timimsms) [inline src/pages/Profile/subpages/Settings/Basics.js:126]: not sure if we meant to leave this commented out

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -7,14 +7,14 @@ import SiteLogo from "./subcomponents/SiteLogo";
...
 export default function Header({ userLoggedIn, avatarIndex, color }) {
   let [showHamburgerDropdown, handleHamburgerClick] = useState(false);
-  let [formIsOpen, setFormIsOpen] = useState(userLoggedIn || window.innerWidth < 1200 ? false : true);
+  let [formIsOpen, setFormIsOpen] = useState(
+    userLoggedIn || window.innerWidth < 1200 ? false : true
+  );
 
-  function filler ({som}) {
-  }
+  function filler() {}
```
Comment (timimsms) [inline src/components/Header/Header.js:17]: if we're not using this for anything, may make sense to remove or just pass an anonymous blank function to `handleHamburgerClick`. E.g., `handleHamburgerClick={() => {}}`;

If we're just using as a placeholder blank/empty function call for now, I think that's fine 👍

**Pull request merged**